common/phish_detector.py

```diff
@@ -172,6 +174,9 @@ def prep_features(self, directory):
 
 
     def generate_features(self):
+        if self.use_name_in_from != 0:
+            Detector.USE_NAME = True
```

Comment:
Where is Detector.USE_NAME being used in phish_detector.py other than setting Detector.USE_NAME = True?